.github/workflows/pull-request-test.yml

```
@@ -0,0 +1,93 @@
+          path: |
+            src/**/__diff_output__/
+          retention-days: 2
+
+      - name: Find PR number
+        if: failure() && github.event_name == 'pull_request'
+        id: pr
+        run: echo "PR_NUMBER=${{ github.event.pull_request.number }}" >> $GITHUB_ENV
+
+      - name: Comment on PR with failure details
+        if: failure() && github.event_name == 'pull_request'
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)